libs/platform/user/ebpf_platform_user.cpp

```diff
@@ -243,26 +249,78 @@ _get_environment_variable(const std::string& name)
     return value;
 }
 
+/**
+ * @brief Get an environment variable as a boolean.
+ *
+ * @param[in] name Environment variable name.
+ * @return true Environment variable is set to "true" or "1".
+ * @return false Environment variable is set to "false", "0", or if it's not set.
+ */
+static bool
+_get_environment_variable_as_bool(const std::string& name)
+{
+    std::string value = _get_environment_variable_as_string(name);
+    if (value.empty()) {
+        return false;
+    }
+
+    // convert value to lower case
+    std::transform(value.begin(), value.end(), value.begin(), [](unsigned char c) { return (char)std::tolower(c); });
+    if (value == "true") {
+        return true;
+    }
+    if (value == "false") {
+        return false;
+    }
+    if (value == "1") {
+        return true;
+    }
+    if (value == "0") {
+        return false;
+    }
+    throw std::runtime_error("Invalid value for environment variable " + name);
```

Comment:
Do we really want this to throw?  According to codecov there's no test case that covers this so I'm a bit concerned by this behavior even though it's only the UM tests.  Maybe it should return true in this case.